tests/test_lightningd.py

```diff
@@ -190,6 +190,19 @@ def fund_channel(self,l1,l2,amount):
         l1.daemon.wait_for_log('-> CHANNELD_NORMAL')
         l2.daemon.wait_for_log('-> CHANNELD_NORMAL')
 
+    def pay(self,lsrc,ldst,amt,label):
```

Comment:
And we might want to have `label=None` and generate a random one if we don't go looking for it later.